fix #105716 Album option Page/Section Break

ericfont opened the pull request.

Jojo-Schmitz commented on mscore/albummanager.ui:
```diff
@@ -196,7 +244,6 @@
   <tabstop>createNew</tabstop>
   <tabstop>load</tabstop>
   <tabstop>print</tabstop>
-  <tabstop>createScore</tabstop>
```
why taking it out of the tabstops? I guess instead checkBoxAddPageBreak and checkBoxAddSectionBreak should get added.

ericfont commented:
@Jojo-Schmitz I've re-added Join Scores to tabstop and added checkBoxAddSectionBreak checkBoxAddPageBreak to tabstop.  

Thanks for catching that.  I guess when I moved the button inside the QGroupBox, the QtCreator ui editor unintentionally removed the tabstop.

ericfont commented on libmscore/score.cpp:
```diff
@@ -2295,11 +2295,19 @@ bool Score::appendScore(Score* score)
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      if (addPageBreak) {
+            // add PageBreak if one doesn't already exist
+            if (!lastMeasure->pageBreak())
```
I'm looking over this line and am thinking maybe I should do <pre>if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak())</pre>instead, so that only a Line or Page break will be the result, not both Line & Page break.

Jojo-Schmitz replied:
So score that have a line break at their end, where this presumably has been added on purpose, don't get a page break. Yes, sounds good to me.

Jojo-Schmitz replied:
OTH if the user specified 'add page break' in the dialog (s)he should get it, shouldn't (s)he?
But you can't have a line break and a page break at the same place, at least not via the palette, so if anything, the page break should **replace ** the line break.

ericfont replied:
Good point...I should modify this so that Page Break will replace preexisting Line Break.

Jojo-Schmitz commented:
rebase needed

ericfont commented on libmscore/score.cpp:
```diff
@@ -1761,13 +1761,17 @@ bool Score::appendScore(Score* score)
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      // apply Page/Section Breaks if desired
```
@Jojo-Schmitz, I've changed around this logic regarding addPageBreak so is as follows.  If adding a pagebreak, will first remove a line break if one exists.  If not adding a page break, will simply add a line break (if a page break or line break doesn't already exist)...this is same behavior pre-PR.

ericfont commented:
(rebased).
